From 2b4ccdbf85e4e5ca5d73050dca53ef9d0f5177d9 Mon Sep 17 00:00:00 2001
From: Alexander Martinz <amartinz@shiftphones.com>
Date: Fri, 27 Aug 2021 13:37:54 +0200
Subject: [PATCH] init: reboot to recovery on panic by default to allow reading
 pstore

Note: bootloaders can still change the reboot target using cmdline.

Change-Id: I982ddb0b5720c660026e9b8b2ec4d0c3196b6442
Signed-off-by: Alexander Martinz <amartinz@shiftphones.com>
Signed-off-by: neobuddy89 <neobuddy89@gmail.com>
Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>
---
 init/reboot_utils.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init/reboot_utils.cpp b/init/reboot_utils.cpp
index 1f411636c532..4df96a95d1db 100644
--- a/init/reboot_utils.cpp
+++ b/init/reboot_utils.cpp
@@ -36,7 +36,7 @@
 namespace android {
 namespace init {
 
-static std::string init_fatal_reboot_target = "bootloader";
+static std::string init_fatal_reboot_target = "recovery";
 static bool init_fatal_panic = false;
 
 // this needs to read the /proc/* files directly because it is called before
